From 0161f94961e34010ca07d4af656677c6daaf53f5 Mon Sep 17 00:00:00 2001
From: Jonathan M Hsieh <jon@cloudera.com>
Date: Thu, 11 Feb 2016 17:52:28 -0800
Subject: [PATCH 193/197] CDH-37191 org.apache.hadoop.hbase.spark.TestJavaHBaseContext unit tests failing on cdh5-1.2.0

- Remove JavaSparkContext.addJar("spark.jar") - causes unit test failure

Change-Id: I56470093265c19767cecaa02eae61a80e4d45102
Reason: Bug
Author: Joanthan Hsieh
Ref: CDH-37191
---
 .../hadoop/hbase/spark/TestJavaHBaseContext.java   |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/hbase-spark/src/test/java/org/apache/hadoop/hbase/spark/TestJavaHBaseContext.java b/hbase-spark/src/test/java/org/apache/hadoop/hbase/spark/TestJavaHBaseContext.java
index 724ac36..6042998 100644
--- a/hbase-spark/src/test/java/org/apache/hadoop/hbase/spark/TestJavaHBaseContext.java
+++ b/hbase-spark/src/test/java/org/apache/hadoop/hbase/spark/TestJavaHBaseContext.java
@@ -65,7 +65,6 @@ public class TestJavaHBaseContext implements Serializable {
   @Before
   public void setUp() {
     jsc = new JavaSparkContext("local", "JavaHBaseContextSuite");
-    jsc.addJar("spark.jar");
 
     File tempDir = Files.createTempDir();
     tempDir.deleteOnExit();
-- 
1.7.0.4

